lib/matplotlib/tests/test_widgets.py

```diff
@@ -1125,6 +1125,16 @@ def test_range_slider(orientation):
 
     fig, ax = plt.subplots()
 
+    slider = widgets.RangeSlider(
+        ax=ax, label="", valmin=0.0, valmax=1.0, orientation=orientation,
+        valinit=[0.0, 0.0]
+    )
+    box = slider.poly.get_extents().transformed(ax.transAxes.inverted())
+    assert_allclose(box.get_points().flatten()[idx], [0, 0.25, 0, 0.75])
+
+    # Check initial value is set correctly
+    assert_allclose(slider.val, (0.0, 0.0))
```

Comment:
@NickolasGiannatos sorry, I have overlooked your question on whether to include into the existing test or add a new test. In this case I'd go with a dedicated test because
- There's little code sharing between the tests (only the idx order, which could be written more compact as `idx = [1, 0, 3, 2] if orientation == 'vertical' else [0, 1, 2, 3]). So there's little gain.
- The existing test is already quite long and the new test would add 10 unrelated lines. This makes understanding what's happening in the existing test much harded. If it was just a 2-3 liner like the slider.reset() below, it'd maybe ok to add to the existing test.